tasmota/tasmota_xnrg_energy/xnrg_12_solaxX1.ino

```diff
@@ -570,15 +556,24 @@ void solaxX1_Show(bool json)
 
 #ifdef USE_WEBSERVER
   } else {
-    WSContentSend_PD(HTTP_SNS_solaxX1_DATA1, solar_power, pv1_voltage, pv1_current, pv1_power);
+    String table_align = Settings->flag5.gui_table_align?"right":"left";
+    WSContentSend_PD(HTTP_SNS_solaxX1_Num, D_SOLAX_X1 " " D_SOLAR_POWER, table_align.c_str(), solar_power, D_UNIT_WATT);
```

Comment:
Now compare the code size when inserting D_SOLAX_X1 and the " " in HTTP_SNS_solaxX1_Num instead of duplicating '%s'

Reply:
Oh yes. Thanks for the hint!